src/main/java/org/sonarsource/sonarlint/ls/SonarLintLanguageServer.java

```diff
@@ -119,10 +121,10 @@
   private static final String SONARLINT_SOURCE = "sonarlint";
   private static final String SONARLINT_OPEN_RULE_DESCRIPTION_COMMAND = "SonarLint.OpenRuleDesc";
   private static final String SONARLINT_DEACTIVATE_RULE_COMMAND = "SonarLint.DeactivateRule";
-  static final String SONARLINT_UPDATE_PROJECT_BINDING_COMMAND = "SonarLint.UpdateProjectBinding";
+  static final String SONARLINT_UPDATE_ALL_BINDINGS_COMMAND = "SonarLint.UpdateAllBindings";
```

Comment:
Nitpicking: this is an API change (and hopefully no-one except SLVSCODE relies upon this API :sweat_smile:)